assets/src/edit-story/components/canvas/draggablePage.js

```diff
@@ -22,13 +22,14 @@ import { forwardRef, useCallback } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+/**
+ * External dependencies
```

Comment:
Check the import order here. This, together with the `prop-types` import should be move up so it's before the `Internal dependencies` comment.